main.rb

```diff
@@ -1,113 +1,10 @@
-require_relative 'item'
```

Comment:
We will need to rewrite this code as it is needed for the whole main to work in the main.rb

Reply:
Ok. thanks for the review